build_tools/python/e2e_test_framework/serialization_test.py

```diff
@@ -81,25 +80,31 @@ def test_serialize_and_pack(self):
     self.maxDiff = None
     self.assertEqual(
         results, {
-            "main_obj": [
-                collections.OrderedDict(
-                    b_list=["id_a", "id_b"],
-                    c_obj=collections.OrderedDict(float_val=0.1),
-                    str_val="test1",
-                    enum_val="OPTION_B"),
-                collections.OrderedDict(
-                    b_list=["id_a"],
-                    c_obj=collections.OrderedDict(float_val=0.2),
-                    str_val=None,
-                    enum_val="OPTION_C")
-            ],
-            "obj_map":
-                collections.OrderedDict({
-                    "test_b:id_a":
-                        collections.OrderedDict(key="id_a", int_val=10),
-                    "test_b:id_b":
-                        collections.OrderedDict(key="id_b", int_val=20)
-                })
+            "main_obj": [{
```

Comment:
Note that you could still use the dict constructor. IDK which is preferred, though the constructor creates fewer quotes as long as all keys are valid identifiers

Reply:
SG. Used in some places to shorten the code.